starport/services/networkbuilder/networkbuilder.go

```diff
@@ -182,7 +190,10 @@ func (b *Builder) Init(ctx context.Context, chainID string, source SourceOption,
 		return nil, err
 	}
 
-	o := newInitOptions(chainID, options...)
+	o, err := newInitOptions(chainID, options...)
```

Comment:
With the option above this stay as `o := newInitOptions(chainID, options...)`.

Reply:
I didn't get your comment